crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -1025,6 +1032,40 @@ fn events(
 
 // ----------------------------------------------------------------------------
 
+fn ime_enabled_filter_events(events: Vec<Event>) -> Vec<Event> {
+    let mut filter_events: Vec<Event> = Vec::new();
+
+    for event in events {
+        match event {
+            Event::Key { repeat: true, .. }
+            | Event::Key {
+                key: Key::Backspace,
+                ..
+            } => {}
```

Comment:
Explain why in a comment

Reply:
no, explain it IN THE CODE. It is the code that will last.

Reply:
> no, explain it IN THE CODE. It is the code that will last.

It has been modified.